### PR TITLE
expose FileReader, for Blob to ArrayBuffer conversion

### PR DESCRIPTION
**Pull request by trevj**
Exposes `FileReader`, for Blob -> ArrayBuffer conversion.

### CODE REVIEW

File: src/entry.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ if (typeof Components !== 'undefined') {
   // See: https://bugzilla.mozilla.org/show_bug.cgi?id=1007318
   Blob = hiddenWindow.Blob;
   WebSocket = hiddenWindow.WebSocket;
+  FileReader = hiddenWindow.FileReader;
```
Comment (willscott) [inline src/entry.js]: Also needs to be added to ```src/spec.js``` if we want to have any hope of testing this?

Comment (trevj) [inline src/entry.js]: I can add that. Should I also be adding a test in the freedom repo which will fail without this?

Comment (willscott) [inline src/entry.js]: Ideally, an rtc test that sends binary data, causing the conversion to
happen :)

Comment (trevj) [inline src/entry.js]: Sure. I updated `spec.js` and added a binary test to this pull request:
https://github.com/freedomjs/freedom/pull/203

**Comment by willscott** [comment]
:+1:

**Pull request merged**